lib/fastlane/plugin/test_center/actions/tests_from_xctestrun.rb

```diff
@@ -114,6 +114,12 @@ def self.available_options
             is_string: false,
             default_value: false,
             optional: true
+          ),
+          FastlaneCore::ConfigItem.new(
```

Comment:
See the [comments](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/pull/258/files#diff-b1627ddcb767b566591f53bc2ae1fcffR362) I made about this option in `multi_scan`.